File: core/src/main/java/platformer/service/rest/requests/AccountDataDTO.java
package platformer.service.rest.requests;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

import java.util.List;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
public class AccountDataDTO {
    private String username;
    private long accountId;
    private long settingsId;
    private int spawn;
    private int coins;
    private int tokens;
    private int level;
    private int exp;
    private List<String> perks;
    private List<String> items;
}

File: game-service/src/main/kotlin/com/games/mw/gameservice/domain/account/requests/AccountDataDTO.kt
package com.games.mw.gameservice.domain.account.requests

data class AccountDataDTO (
    val username: String,
    val accountId: Long,
    val settingsId: Long,
    val spawn: Int,
    val coins: Int,
    val tokens: Int,
    val level: Int,
    val exp: Int,
    val items: List<String>,
    val perks: List<String>
)